templates/mongodb/mongodb.yaml

```diff
@@ -27,17 +28,13 @@ spec:
       affinity:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      volumes:
-      - name: mongodb-storage
-        persistentVolumeClaim:
-          claimName: {{ template "cytomine.name" . }}
       containers:
         - name: {{ .Chart.Name }}-mongodb
           image: "{{ .Values.mongodb.image.repository }}:{{ .Values.mongodb.image.tag }}"
           imagePullPolicy: {{ .Values.mongodb.image.pullPolicy }}
           command: ["mongod"]
           volumeMounts:
-            - name: mongodb-storage
+            - name: {{ template "cytomine.name" . }}
```

Comment:
Templating names are not wrong, but here I think it's overkill. If used, it definitely needs to be suitable which it's not here, needs to be concatenated with something.

Reply:
Looking further, it seems main is also broken, so that should be fixed. But that's also a good example of a bug that probably wouldn't have crept in if one templated less :)